src/direct-linking/background/storage.test.ts

```diff
@@ -0,0 +1,155 @@
+import indexedDB from 'fake-indexeddb'
+import IDBKeyRange from 'fake-indexeddb/lib/FDBKeyRange'
+import normalize from '../../util/encode-url-for-id'
+
+import Storage from '../../search/search-index-new/storage'
+import { StorageManager } from '../../search/search-index-new/storage/manager'
+// import { AnnotationStorage } from './storage'
+import AnnotationBackround from './'
+
+import * as dummy from './storage.test.data'
+
+const runSuite = () => {
+    const storageManager = new StorageManager()
+    const annotationStorage = new AnnotationBackround({ storageManager })
+        .annotationStorage
+
+    async function insertTestData() {
+        // Insert annotations and direct links
+        annotationStorage.insertDirectLink(dummy.directLink)
```

Comment:
Each of these method calls are async ops. To ensure the `await insertTestData()` line in `resetTestData` function works as intended (resolves once all data is guaranteed to be inserted), you should either put all these method calls in a `Promise.all` call (to run them simultaneously) or put `await` in front of each call (to run them sequentially).

Reply:
Ah yeah, forgot to add await in front of them!